project_task_code/views/project_view.xml

```diff
@@ -8,7 +8,7 @@
             <field name="arch" type="xml">
                 <field name="name" position="before">
                     <field name="code" class="oe_inline" />
-                    <label string=" - " class="oe_inline" />
+                    <label for=" - " class="oe_inline" />
```

Comment:
It's referring to string selectors, which means an [Xpath](http://www.odoo.yenthevg.com/xpath-expressions-in-odoo-8/) targetting a string attribute such as this one. The string attributes themselves aren't bad, just the use of the string as an Xpath injection point when inheriting.

Along those lines it is good to add name attributes to important objects like notebook pages that only have strings in order to allow for easier targeting. Not applicable here though 😄